app/poros/food.rb

```diff
@@ -0,0 +1,18 @@
+class Food
+  attr_reader :code,
+              :description,
+              :brand,
+              :ingredients
+
+
+  def initialize(data)
+    @code = data[:fgtinUpc]
+    @description = data[:description]
+    @brand = data[:brandOwner]
+    @ingredients = data[:ingredients]
+  end
+
+  def results_count
+    @results_count
+  end
```

Comment:
It doesn't look like this is being used- it's probably sufficient to let FoodsFacade give us this info